src/biogeophys/TemperatureType.F90

```diff
@@ -156,6 +156,7 @@ subroutine Init(this, bounds, &
     real(r8)          , intent(in) :: em_perroad_lun(bounds%begl:)
     logical           , intent(in) :: is_simple_buildtemp  ! Simple building temp is being used
     logical           , intent(in) :: is_prog_buildtemp    ! Prognostic building temp is being used
+    real(r8)          , intent(in) :: exice_init_stream_col(bounds%begc:)  ! initial excess ice concentration from the stream file
```

Comment:
The one thing I wonder about though is the name -- I don't get a nice sense of what quantity it is by the name. Maybe include concentration (or conc) in the name? 

I'd also suggest removing "stream" from the name as it will still be set to something without streams.

Reply:
Changed to exice_init_conc_col